app/controllers/application_controller.rb

```diff
@@ -29,4 +35,17 @@ def accessible_hosts
   def known_host?
     accessible_hosts.include?(request.host) || accessible_hosts.include?(request.host_with_port)
   end
+
+  def not_found
+    render_404
+  end
+
+  def render_404
```

Comment:
Given `not_found` only calls `render_404` why not move the body of `render_404` inside `not_found` and remove `render_404`?